go/database/mpt/nibble.go

```diff
@@ -64,6 +64,24 @@ func KeyToNibblePath(key common.Key, source NodeSource) []Nibble {
 	return res
 }
 
+// addressToHashedPath converts the given path into a slice of Nibbles.
+// It always hashes the path before converting it.
+func addressToHashedPath(address common.Address) []Nibble {
```

Comment:
The other functions in this file refer to `[]Nibble` as a `NibblePath` since `Path` is a different type in this package.

Reply:
I do not understand this comment